packages/flutter/lib/src/material/material_state.dart

```diff
@@ -257,22 +266,47 @@ abstract class MaterialStateMouseCursor extends MouseCursor implements MaterialS
   /// disabled, the cursor resolves to [SystemMouseCursors.basic].
   ///
   /// This cursor is the default for many Material widgets.
-  static const MaterialStateMouseCursor clickable = _ClickableMouseCursor();
+  static const MaterialStateMouseCursor clickable = _SpecifyDisabledMouseCursor(
+    usualCursor: SystemMouseCursors.click,
+    disabledCursor: SystemMouseCursors.basic,
+    name: 'clickable',
+  );
+
+  /// A mouse cursor for material widgets related to text, which resolves differently
+  /// when the widget is disabled.
+  ///
+  /// By default this cursor resolves to [SystemMouseCursors.text]. If the widget is
+  /// disabled, the cursor resolves to [SystemMouseCursors.basic].
+  ///
+  /// This cursor is the default for many Material widgets.
+  static const MaterialStateMouseCursor textable = _SpecifyDisabledMouseCursor(
+    usualCursor: SystemMouseCursors.text,
+    disabledCursor: SystemMouseCursors.basic,
+    name: 'textable',
+  );
 }
 
-class _ClickableMouseCursor extends MaterialStateMouseCursor {
-  const _ClickableMouseCursor();
+class _SpecifyDisabledMouseCursor extends MaterialStateMouseCursor {
+  const _SpecifyDisabledMouseCursor({
+    this.usualCursor,
```

Comment:
This is just bike-shedding, but it feels like "enabledCursor" would be the natural name for this